src/pages/developers/index.tsx

```diff
@@ -136,6 +139,42 @@ const StyledCallout = chakra(Callout, {
   },
 })
 
+const SpeedRunEthereumBanner = ({
+  title,
+  linkLabel,
+}: {
+  title: string
+  linkLabel: string
+}) => (
+  <Box position="relative" mb={{ xl: 12 }}>
+    <Image
+      src={SpeedRunEthereumImage}
+      alt="SpeedRunEthereum banner"
+      sizes="100vw"
+      style={{ width: "100vw", objectFit: "cover", objectPosition: "20%" }}
+      h={{
+        base: "450px",
+        xl: "auto",
+      }}
+    />
+    <Stack
+      spacing={{ base: "3", md: "4" }}
+      p={{ base: "6", lg: "8" }}
+      position="absolute"
+      insetInlineStart={{ md: "8" }}
+      maxW={{ base: "lg", xl: "xl" }}
+      top={{ base: "0", md: "50" }}
+      wordBreak="break-word"
+      alignItems="flex-start"
+    >
+      <Heading as="h2" size="xl">
```

Comment:
This is actually the default in Chakra, so you do not need to pass props here to render an h2 with `xl` size.

```suggestion
      <Heading>
```

We will also be handling more defaults under the hood soon to minimize the need for passing a value to `size`

Reply:
oh nice, just saw the component description: 

![image](https://github.com/ethereum/ethereum-org-website/assets/2486142/8935a677-ad95-4ee1-8ee3-c14e9a0f7409)

Thanks!!